pyfixest/estimation/feols_compressed_.py

```diff
@@ -184,6 +184,8 @@ def prepare_model_matrix(self):
         # overwrite Y, X, _data
         self._data_long = data_long_mundlak if self._use_mundlak else data_long
         self._Yd = compressed_dict.Y.to_pandas()
+        # store compressed dependent variable before demeaning
+        self._Y_untransformed = self._Yd.copy()
```

Comment:
This is only valid because the compression algo does not demean, right?

Reply:
My idea was that `prepare_model_matrix` (where `_Y_untransformed` is set) will always be called before demean. This is also the case for `feols`: https://github.com/py-econometrics/pyfixest/blob/8435537ade9ce3e02702c4f63603e9801cc46afe/pyfixest/estimation/FixestMulti_.py#L281-L282

The reason that I had to add this line for `feols_compressed` is that the call to `super().prepare_model_matrix` stores `_Y_untransformed` before compression whereas we need the dependent variable before demeaning but after compression for prediction.

Reply:
I think this should be ok. We need to assign Y_untransformed via the "long" / "uncompressed" dependent variable because we want to make predictions based on the uncompressed variable. Hence if we call `.predict()` on the compressed matrix, it will use Y_untransformed when `newdata = None`.